crates/bevy_app/Cargo.toml

```diff
@@ -27,7 +27,7 @@ ron = { version = "0.6.2", optional = true }
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 wasm-bindgen = { version = "0.2" }
-web-sys = { version = "0.3", features = [ "Window" ] }
+web-sys = { version = "=0.3.51", features = [ "Window" ] }
```

Comment:
Why are you forcing a specific version? This will cause an opaque unresolvable dependency resolution error if any dependency depends on a different patch version.

Reply:
`web-sys = "=0.3.51"` is forced by wgpu. This bug is caused by `resolver = 2` which is needed by wgpu. Maybe this will be resolved after switching to the 2021 edition.